nvflare/app_opt/lr/fedavg.py

```diff
@@ -0,0 +1,222 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
should we put this file in app_opt/lr?
or app_common/np?
Since no additional package needs to be installed